packages/tab/src/Tabs.js

```diff
@@ -32,15 +33,17 @@ const TabItems = ({
   );
 
   const TABs = tabItems.map((tab, index) => (
-    <Tab
-      index={index}
-      label={
+    <ToolTip {...tab.tooltipStyles} title={tab.toolTipText || '.'} arrow placement="top">
```

Comment:
@Senait7-ux Can we have configuration to Enable or Disable tooltip?

Reply:
This has now been resolved, i just pushed the latest code